src/clients/c++/concurrency_manager.cc

```diff
@@ -0,0 +1,519 @@
+// Copyright (c) 2018-2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
For new files, the copyright year should just be 2019

Reply:
Done